vulnerabilities/models.py

```diff
@@ -123,3 +128,36 @@ class PackageReference(models.Model):
 
     def __str__(self):
         return self.platform
+
+
+class Importer(models.Model):
+    """
+    Metadata and pointer to the implementation for a source
+    of vulnerability data (aka security advisories)
+    """
+    name = models.CharField(max_length=100, unique=True, help_text='Name of the importer')
+    license = models.CharField(max_length=100, blank=True, help_text='License of the vulnerability data')
+    last_run = models.DateTimeField(null=True, help_text='UTC Timestamp of the last run')
+    
+    data_source = models.CharField(
+        max_length=100,
+        unique=True,
+        help_text='Class name of the data source implementation importable from  vulnerabilities.importers',
+    )
+    data_source_cfg = pgfields.JSONField(
+        null=False,
+        default=dict,
+        help_text='Implementation-specific configuration for the data source',
+    )
+
+    def make_data_source(self, cutoff_date=None) -> DataSource:
+        """
+        Return a configured and ready to use instance of
+        this importers data source implementation.
+
+        cutoff_date - timestamp of the oldest data to include in the import (default: self.last_run)
+        """
+        cd = cutoff_date or self.last_run
+        importers_module = importlib.import_module('vulnerabilities.importers')
+        cls = getattr(importers_module, self.data_source)
```

Comment:
```
 cls = getattr(importers_module, self.data_source)
 return cls(cutoff_date=cd, config=self.data_source_cfg)
```
 `cls` is a python keyword ,IMHO to avoid confusion, if this variable can be renamed please consider renaming it.

Reply:
It's not a keyword, but a name conventionally used in one specific case and this is not it. I'll rename it to `klass`.